spec/models/profile_photo_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+feature "profile photo" do
+  scenario "user uploads a profile photo" do
+    visit root_path
+    click_link "Sign Up"
+
+    fill_in "Email", with: "ash@s-mart.com"
+    fill_in "Password", with: "boomstick!3vilisd3ad"
+    fill_in "Password confirmation", with: "boomstick!3vilisd3ad"
+    attach_file :user_profile_photo, "#{Rails.root}/spec/support/images/photo.png"
+    click_button "Sign up"
+
+    expect(page).to have_content("Welcome! You have signed up successfully.")
+    
```

Comment:
This would be a good place to check to see whether the image displays (assuming that your redirected page shows the photo somewhere) -- `expect(page).to have_css("img[src*='photo.png']")`